### ISSUE TITLE
Java

### ISSUE THREAD

**Issue by YangShuogeng** [nus-tic4002-AY2122s2/forum#51]
Hi Prof.,

Could you help to see why the below errors happen and the solution?

Java code GitHub address: https://github.com/YangShuogeng/tp2/tree/v1.4_Update_Email_Feature

Email command: type `email NAME`
For example: `email micheal`. If the name `micheal` is matched, then the email window will show.  

I created an email feature in our tp. The new email window is able to show, but the function to send the email does not work as show two errors shown below when I click the send button:

Email window:

![Email Window](https://user-images.githubusercontent.com/54273624/162551837-ddb167f6-d0a8-4032-b36f-ae9714f356ed.png)

Send button errors:

![Email Send button error](https://user-images.githubusercontent.com/54273624/162551850-0c8caf92-1ce9-4f9b-acc6-db532720015f.png)

Another error is about the new window after the email sends successfully as shown below (I omitted the email send feature first to test the new window):

 
![Send button without sending feature](https://user-images.githubusercontent.com/54273624/162552012-80c026a4-1599-496f-8c06-825b89dc9be5.png)

**Comment by damithc** [comment]
Perhaps this is the cause?
![image](https://user-images.githubusercontent.com/1673303/162555194-c9ddb947-ea6c-46d5-88ea-3ba7e4dcaf1a.png)

**Comment by YangShuogeng** [comment]
> Perhaps this is the cause? ![image](https://user-images.githubusercontent.com/1673303/162555194-c9ddb947-ea6c-46d5-88ea-3ba7e4dcaf1a.png)

Hi Prof.,

Thanks~ That NullPointerException bug was fixed.

Do you have any idea about the first one?

**Comment by damithc** [comment]
> Do you have any idea about the first one?

@YangShuogeng Not really. This feature is something new you are adding, which means debugging is part of the deliverables :-)

Some general info that might help https://www.baeldung.com/java-lang-reflect-invocationtargetexception
